qiskit_dynamics/array/__init__.py

```diff
@@ -57,53 +56,49 @@
     # Apply numpy ufuncs
     np.cos(a) + 1j * np.sin(a)
 
-For the JAX Array backend, only NumPy functions that have a corresponding function
-in the ``jax`` library that has been registered with the dispatch module can be
-applied to the functions. Trying to apply an unsupported ``numpy`` function to
-these arrays will raise an exception.
+For the JAX Array backend, only Numpy functions that have a corresponding function in the ``jax``
+library that has been registed with the dispatch module can be applied to the functions. Trying to
+apply an unsupported ``numpy`` function to these arrays will raise an exception.
 
 Default Backend
 ---------------
 
-When initializing a new :class:`Array`, the ``backend`` kwarg is used to specify
-the array backend. This will convert the input data to an array of this backend
-if required. If ``backend=None``, the default backend will be used.
-The initial default backend is always set to ``"numpy"``.
+When initializing a new :class:`Array`, the ``backend`` kwarg is used to specify which array backend
+to use. This will convert the input data to an array of this backend if required. If
+``backend=None``, the default backend will be used. The initial default backend is always set to
+``"numpy"``.
 
-The current default backend can be viewed by using the class method
-:meth:`Array.default_backend`. A different
-default backend can be set for all :meth:`Array` instances by using the
+The current default backend can be viewed by using the class method :meth:`Array.default_backend`. A
+different default backend can be set for all :meth:`Array` instances by using the
 :meth:`Array.set_default_backend` class method.
 
 Attributes and Methods
 ----------------------
 
-The :class:`Array` class exposes the same methods and attributes of the wrapped
-array class and adds two additional attributes
+The :class:`Array` class exposes the same methods and attributes of the wrapped array class and adds
+two additional attributes:
 
 * :attr:`~Array.data` which returns the wrapped array object
 * :attr:`~Array.backend` which returns the backend string of the wrapped array.
 
-All other attributes and methods of the wrapped array are accessable
-through this class, but with any array return types wrapped into
-:class:`Array` objects. For example
+All other attributes and methods of the wrapped array are accessible through this class, but with
+any array return types wrapped into :class:`Array` objects. For example:
 
 .. jupyter-execute::
 
     # Call an numpy.ndarray method
     a.reshape((2, 5))
 
 
-
 Array Initialization
 --------------------
 
-An :class:`Array` object can be initialized as ``Array(data)`` from any ``data`` that is
+An :class:`Array` object can be initialized as ``Array(data)`` from any ``data`` that is:
 
-1. An ``array`` object of one of the supported backends.
+1. An :class:`Array` object of one of the supported backends.
```

Comment:
I think this shouldn't be
```
:class:`Array`
```
as this sentence is referring to arrays in the underlying array libraries, not the wrapping `Array` class of this module. I think either change back to
```
``array``
```
or just the word "array" with no markup.